concent_api/core/tests/test_auth_accept_or_reject_results.py

```diff
@@ -425,24 +430,21 @@ def test_requestor_sends_subtask_results_accepted_and_concent_should_return_it_t
         self._assert_stored_message_counter_not_increased()
 
         # STEP 4:
-        # 4.1. TaskToCompute is send signed with different key, request is rejected with proper error message.
-        # 4.2. TaskToCompute is send with different requestor public key, request is rejected with proper error message.
-        # 4.3. TaskToCompute is send with different data, request is rejected with proper error message.
+        # ReportComputedTask is send signed with different key, request is rejected with proper error message.
 
-        # 4.1.
-        task_to_compute.sig = None
-        task_to_compute = self._sign_message(
-            task_to_compute,
-            self.DIFFERENT_REQUESTOR_PRIVATE_KEY,
+        report_computed_task.sig = None
```

Comment:
I am not sure if I understood the diff correctly, but didn't we now lost the test for missing signature in TTC in favor of missing signature in RCT?

Reply:
yes. Some scenarios are feasible anymore - due to the fact that now:
SRA (requestor signed) -> **RCT** (provider signed) -> TTC (requestor signed)

If TTC didn't have signature it would mean that Provider had sent such a message (in RCT) and Requestor had accepted it.